engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceModificationProcessor.java

```diff
@@ -81,6 +82,12 @@
       with a scope element that doesn't belong to the activating element's flow scope. \
       These variables should be set before or after the modification.""";
 
+  private static final String ERROR_MESSAGE_MORE_THAN_ONE_FLOW_SCOPE_INSTANCE =
+      """
+      Expected to modify instance of process '%s' but it contains one or more activate instructions \
+      for an element that has a flow scope with more than one active instances. Can't decide in \
```

Comment:
```suggestion
      for an element that has a flow scope with more than one active instance. Can't decide in \
```

🔧 I believe this should be singular.